.github/workflows/auto-docs.yml

```
@@ -0,0 +1,160 @@
+      - name: Comment when skipped
+        if: steps.guardrail.outputs.skip == 'true'
+        run: |
+          reason=$(tail -n +2 pr-body.md)
+          gh issue comment "${{ github.event.issue.number }}" --body "Auto-docs pipeline ran but made no changes: ${reason:-no docs update was needed for this ticket.}"
+          rm -f pr-body.md
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
+      - name: Set up Node
+        if: steps.guardrail.outputs.skip == 'false'
+        uses: actions/setup-node@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)